### ISSUE TITLE
Split of KITTY dataset for task2

### ISSUE THREAD

**Issue by Ivancaminal72** [ComputerVisionCentre/mcv-m5#4]
Hi,
In task 2 the test split of the KITTY dataset is empty because at the end of the script.py (that is inside the KITTY directory) the call for the 'proces_set' function that generates the test directory is commented. How we can evaluate it?
Thanks in advance

**Comment by lluisgomez** [comment]
The test set of the KITTI dataset is not public. You should provide only results on the training and validation sets.
